### PR TITLE
RHMAP-6497 download file

### PR DESCRIPTION
**Pull request by pb82**
Use the `fhc appdata download` command to download the exported file.

No version bump because one version bump for the whole feature branch should be enough.

ping @odra

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* There should be an associated fhcap PR with this change - please provide a link to this
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.

In order to update fhcap with your component change please execute:
`fhcap component update fh-fhc 2.7.4 316`

File: lib/cmd/fh3/appdata.js [hunk 1]
```diff
@@ -142,15 +142,20 @@ function downloadAppData(args, cb) {
...
 
-    return cb(null, 'Download URL: ' + res.url);
-  }
+    var params = {};
+    params.url = res.url;
+    params.method = 'GET';
+    params.output = downloadFilePath;
+
+    // Then download the file
+    fhreq.downloadFile(params, cb);
```
Comment (odra) [inline lib/cmd/fh3/appdata.js]: Shouldn't we check if a file already exists in this path?

Comment (pb82) [inline lib/cmd/fh3/appdata.js]: Yes, you're right. Will update the PR.

File: lib/cmd/fh3/appdata.js [hunk 2]
```diff
@@ -142,15 +142,20 @@ function downloadAppData(args, cb) {
...
     if (err) {
       return cb(err);
     }
 
-    return cb(null, 'Download URL: ' + res.url);
-  }
+    var params = {};
+    params.url = res.url;
+    params.method = 'GET';
+    params.output = downloadFilePath;
```
Comment (paolobueno) [inline lib/cmd/fh3/appdata.js]: Should we not allow file overwrite as we do for the forms export?

**Comment by paolobueno** [comment]
LGTM, however shouldn't we use the 'new' command structure with this? i.e. `fh3/appdata/export.js`, `fh3/appdata/list.js`, `fh3/appdata/download.js`, like `fhc admin environments`

We can perhaps do this refactoring on another JIRA.

**Comment by pb82** [comment]
@paolobueno yes we should refactor it using the new command structure. Talked about that with Leonardo and we will do that if there's some time at the end of the sprint.

**Comment by paolobueno** [comment]
@pb82 oh, nice, thanks. I wasn't able to review the previous PRs.

**Comment by odra** [comment]
LGTM

**Pull request merged**